FEM/rf_ic_new.cpp

```diff
@@ -861,27 +861,7 @@ void CInitialCondition::SetDomain(int nidx)
 				return;
 			}
 			std::ifstream rfr_file;
-			std::string restart_file_name;
-			restart_file_name = rfr_file_name;
-			basic_string<char>::size_type indexChWin, indexChLinux;
-			indexChWin = indexChLinux = 0;
-			indexChWin = FileName.find_last_of('\\');
-			indexChLinux = FileName.find_last_of('/');
-			//
-			string funfname;
-			if (indexChWin == string::npos && indexChLinux == string::npos)
-				funfname = rfr_file_name;
-			else if (indexChWin != string::npos)
-			{
-				funfname = FileName.substr(0, indexChWin);
-				funfname = funfname + "\\" + rfr_file_name;
-			}
-			else if (indexChLinux != string::npos)
-			{
-				funfname = FileName.substr(0, indexChLinux);
-				funfname = funfname + "/" + rfr_file_name;
-			}
-			restart_file_name = funfname;
+			std::string restart_file_name = FilePath + rfr_file_name;
```

Comment:
why not use `pathJoin()` in FileTools.h?

Reply:
I had a commit to replace FilePath + name with pathJoin but I removed it soon. With that replacement, "#include "FileTools"" were added in several files and many places were changed.  Later on I thought that using FilePath + name is more straightforward than calling pathJoin.

Reply:
i see